tests/test_wsi_registration.py

```diff
@@ -352,21 +354,54 @@ def test_register_output_without_initializer(
     assert np.linalg.norm(expected[:2, :2] - output[:2, :2]) < 0.1
     assert np.linalg.norm(expected[:2, 2] - output[:2, 2]) < 10
 
+    _ = df.register(
```

Comment:
is this line testing the output?

Reply:
I added it to fix partial test coverage for these lines https://github.com/TissueImageAnalytics/tiatoolbox/blob/625aa3719ab4b7f3d17501e38c65363115f81dc9/tiatoolbox/tools/registration/wsi_registration.py#L1115-L1118